export function handleError(message: string){
    throw new Error(message);
}